src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java

```diff
@@ -176,87 +178,150 @@ private  JPanel buildScaPanel(Result result, @NotNull Runnable runnableDraw, Run
-                packageName.setCursor(new Cursor(Cursor.HAND_CURSOR));
-                packageName.addMouseListener(new MouseAdapter() {
+
+        JLabel remediation = new JLabel();
+        remediation.setBorder(JBUI.Borders.empty(0, 10, 20, 0));
+        if (StringUtils.isNotBlank(result.getData().getRecommendedVersion())) {
+            remediation.setIcon(AllIcons.Diff.MagicResolve);
+            remediation.setToolTipText(Bundle.message(Resource.AUTO_REMEDIATION_TOOLTIP));
+            remediation.setText(String.format(Constants.HTML_FONT_YELLOW_FORMAT, hex, "Upgrade to version: " + result.getData().getRecommendedVersion()));
+            remediation.setCursor(new Cursor(Cursor.HAND_CURSOR));
+            if (result.getData().getScaPackageData().isSupportsQuickFix()) {
+                remediation.addMouseListener(new MouseAdapter() {
                     @SneakyThrows
                     @Override
                     public void mouseClicked(MouseEvent e) {
-                        Desktop.getDesktop().browse(new URI(packageData.getUrl()));
+                        handleGeneralRemediationClick(remediation, result);
                     }
                 });
-                linksPanel.add(packageName, "gapleft 6");
+            } else {
+                remediation.setEnabled(false);
             }
-            scaBody.add(linksPanel, "wrap, split 2");
         } else {
-            scaBody.add(new JLabel("No information"), "span, growx, gapbottom 5");
+            remediation.setText("No information");
         }
-        JBScrollPane scrollPane = new JBScrollPane(scaBody);
-        scrollPane.setHorizontalScrollBarPolicy(JScrollPane.HORIZONTAL_SCROLLBAR_NEVER);
-        details.add(scrollPane, "span, growx, wrap");
 
-        return details;
+        scaBody.add(remediation);
     }
 
-    @NotNull
-    private JPanel getSCAVulnerabilityPathPanel(Result result, JPanel scaBody) {
-        JLabel vulnerabilityPathTitle = new JLabel(String.format("<html><b>%s</b></html>", boldLabel(Bundle.message(Resource.PATH)).getText()));
-        scaBody.add(vulnerabilityPathTitle, "span, growx");
-        List<List<DependencyPath>> dependencyPaths;
-        dependencyPaths = result.getData().getScaPackageData() != null ? result.getData().getScaPackageData().getDependencyPaths() : null;
+    /**
+     * Draw SCA about vulnerability section
+     *
+     * @param scaBody - body panel
+     */
+    private void drawSCAAboutVulnerability(Result result, JPanel scaBody) {
+        addSectionTitle(scaBody, Resource.ADDITIONAL_KNOWLEDGE);
+
+        JLabel aboutVulnerability = new JLabel(String.format(Constants.HTML_WRAPPER_FORMAT, boldLabel(Bundle.message(Resource.ABOUT_VULNERABILITY)).getText()));
+        aboutVulnerability.setBorder(JBUI.Borders.empty(0, 10, 20, 0));
+        aboutVulnerability.setIcon(CxIcons.ABOUT);
+        aboutVulnerability.setText("About this vulnerability");
```

Comment:
Is this needed? Can we use the resource used in line 292?

Reply:
Fixed